testsuite/features/support/env.rb

```diff
@@ -26,13 +26,14 @@
 STARTTIME = Time.new.to_i
 Capybara.default_max_wait_time = ENV['CAPYBARA_TIMEOUT'] ? ENV['CAPYBARA_TIMEOUT'].to_i : 10
 DEFAULT_TIMEOUT = ENV['DEFAULT_TIMEOUT'] ? ENV['DEFAULT_TIMEOUT'].to_i : 250
+$is_cloud_provider = ENV["PROVIDER"].include? 'aws'
 
 # QAM and Build Validation pipelines will provide a json file including all custom (MI) repositories
 custom_repos_path = File.dirname(__FILE__) + '/../upload_files/' + 'custom_repositories.json'
 if File.exist?(custom_repos_path)
   custom_repos_file = File.read(custom_repos_path)
   $custom_repositories = JSON.parse(custom_repos_file)
-  $build_validation = true
+  $build_validation = !$is_cloud_provider
```

Comment:
Are you running the testsuite using a custom_repositories.json file on it? Are you trying to run a BV in cloud?
I really think this value should not be touched in any case. This value will be true if we have a "custom_repositories.json" file in the upload_files folder. No need to worry about being or not in cloud.